### PR TITLE
fix: add words to companies, network-protocol.txt, and software-tools.txt

### PR DESCRIPTION
**Pull request by digger-yu**
# Add Dictionary

Dictionary:network-protocol.txt software-term.txt software-tools.txt

## Description

add words to network-protocol.txt software-term.txt software-tools.txt

There are several words related to NICs, temporarily placed in the  network-protocol.txt file
If there is a more suitable file for them, I will adjust it later

## References

- _Any source references._

## Checklist

- [x] By submitting this issue, you agree to follow our
      [Code of Conduct](https://github.com/streetsidesoftware/cspell-dicts/blob/main/CODE_OF_CONDUCT.md)

### CODE REVIEW

File: dictionaries/software-terms/src/software-terms.txt [hunk 1]
```diff
@@ -1704,3 +1704,4 @@ zip
 zlib
 ZPool
 ZPools
+trex
```
Comment (Jason3S) [inline dictionaries/software-terms/src/software-terms.txt:1707]: @digger-yu,

What is `trex`?

Comment (digger-yu) [reply dictionaries/software-terms/src/software-terms.txt:1707]: A tool from cisco

Comment (Jason3S) [reply dictionaries/software-terms/src/software-terms.txt:1707]: @digger-yu,

Would you please add it to tools instead?

Comment (digger-yu) [reply dictionaries/software-terms/src/software-terms.txt]: @Jason3S  Revision completed

**Comment by Jason3S** [comment]
@digger-yu,

Thank you.

**Pull request merged**